### PR TITLE
AC-566 Explicit types can be replaced with <>

### PR DESCRIPTION
**Pull request by rishabh-997**
## Description of what I changed
Converted all occurrences of "Explicit types can be replaced with <>" and found a single instance of lambda function not being used so updated it too

## Issue I worked on

JIRA Issue: https://issues.openmrs.org/browse/AC-566

## Checklist: I completed these to help reviewers :)

- [x] My pull request only contains **ONE single commit**
(the number above, next to the 'Commits' tab is 1).

- [ ] I have **added tests** to cover my changes. (If you refactored
existing code that was well tested you do not have to add tests)

- [x] All new and existing **tests passed**.

- [x] My pull request is **based on the latest changes** of the master branch.

### CODE REVIEW

File: openmrs-client/src/main/java/org/openmrs/mobile/utilities/ActiveAndroid/Cache.java [hunk 1]
```diff
@@ -71,7 +71,7 @@ public static synchronized void initialize(Configuration configuration) {
         // actually used, however at this point it seems like the reflection
         // required would be too costly to be of any benefit. We'll just set a max
         // object size instead.
```
Comment (saadkhaleeq610) [inline openmrs-client/src/main/java/org/openmrs/mobile/utilities/ActiveAndroid/Cache.java]: Remove Comments

Comment (rishabh-997) [reply openmrs-client/src/main/java/org/openmrs/mobile/utilities/ActiveAndroid/Cache.java]: why exactly??? It is a standard ActiveAndroid Library

File: openmrs-client/src/main/java/org/openmrs/mobile/utilities/ActiveAndroid/util/SQLiteUtils.java [hunk 2]
```diff
@@ -322,7 +322,7 @@ public static String createColumnDefinition(TableInfo tableInfo, Field field) {
                  * Obtain the columns ordered to fix issue #106 (https://github.com/pardom/ActiveAndroid/issues/106)
```
Comment (saadkhaleeq610) [inline openmrs-client/src/main/java/org/openmrs/mobile/utilities/ActiveAndroid/util/SQLiteUtils.java]: Remove Comments as the code itself is self-explanatory.

Comment (f4ww4z) [reply openmrs-client/src/main/java/org/openmrs/mobile/utilities/ActiveAndroid/util/SQLiteUtils.java]: @saadkhaleeq610 Actually this PR is not about removing comments.

**Pull request merged**